charger/em2go.go

```diff
@@ -271,8 +313,31 @@ func (wb *Em2Go) phases1p3p(phases int) error {
 	b := make([]byte, 2)
 	binary.BigEndian.PutUint16(b, uint16(phases))
 
-	_, err := wb.conn.WriteMultipleRegisters(em2GoRegPhases, 1, b)
-	return err
+	//Workaround
+	if wb.workaround {
+		// When enabled, disable, wait 10 seconds, enable and set phases
+		enabled, err := wb.Enabled()
+		if err != nil {
+			return err
+		}
+
+		if enabled {
+			if err := wb.Enable(false); err != nil {
+				return err
+			}
+			time.Sleep(10 * time.Second)
+
+			if err := wb.Enable(true); err != nil {
+				return err
+			}
+		}
+	}
+
+	if _, err := wb.conn.WriteMultipleRegisters(em2GoRegPhases, 1, b); err != nil {
```

Comment:
Should that not better be placed before enabling again?

Reply:
> Should that not better be placed before enabling again?

In general you are right, but the chargers old fw is always starting with 3p, even if we send the command in the first place. That´s why we first need to enable it, then set the phases, This is happening within 60ms, so it should not be an issue for the EV.